src/client/utils/Date.ts

```diff
@@ -3,7 +3,7 @@
  */
 export default class DateUtils {
     public static convertToDate(date: string): Date {
-        if (date === null) return null;
+        if (date == null) return null;
```

Comment:
Keep the type check

Reply:
the type check is because it is now used in `MarketUtils` along with optional chaining, which returns `undefined`.

Reply:
Use nullish coalescent to pass a true null for a better typing